src/app/pages/task/task.page.html

```diff
@@ -53,5 +53,8 @@ <h2>{{ item.title }}</h2>
       <ion-badge color="secondary" *ngIf="online">Online</ion-badge>
       <ion-badge color="primary" *ngIf="!online">Offline</ion-badge>
     </ion-label>
+    <ion-button color="primary" fill="outline" (click)="restart()">
```

Comment:
[Suggestion] We may bring that to the settings screen where we will enable this feature and could also use the toggle button to change status.